.github/workflows/airbyte-ci-tests.yml

```diff
@@ -13,15 +13,54 @@ on:
       - synchronize
     paths:
       - airbyte-ci/**
+    paths-ignore:
+      - '**/*.md'
 jobs:
   run-airbyte-ci-tests:
     name: Run Airbyte CI tests
     runs-on: "conn-prod-xlarge-runner"
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v3
+      - name: Get changed files
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            ops:
+              - 'airbyte-ci/connectors/connector_ops/**'
+            base_images:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/base_images/**'
+            pipelines:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/base_images/**'
+              - 'airbyte-ci/connectors/pipelines/**'
+            metadata-lib:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/metadata/lib/**'
+            metadata-orchestrator:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/metadata/lib/**'
+              - 'airbyte-ci/connectors/metadata/orchestrator/**'
+
+      - name: Run airbyte-ci/connectors/connector_ops tests
+        if: steps.changes.outputs.ops == 'true'
+        id: run-airbyte-ci-connectors-connector-ops-tests
+        uses: ./.github/actions/run-dagger-pipeline
+        with:
+          context: "pull_request"
+          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
+          gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
+          sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
+          github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
```

Comment:
are all these things really needed for the tests? I would be all for reducing the amount of secrets being passed in to only those necessary (if they're not needed)

Reply:
maybe things passed for me because I had these things already set in my env 🤷🏼

Reply:
🤔 most are are needed for a remote run.

1. the action has a docker login
2. we need gcs for logs
3. sentry for errors
4. github token for reporting checks back

The only thing that is suspect is GSM.... Let remove it